backend/ethereum/channel/adjudicator.go

```diff
@@ -105,12 +111,35 @@ func toEthSignedStates(subChannels []channel.SignedState) (ethSubChannels []adju
 		ethSubChannels[i] = adjudicator.AdjudicatorSignedState{
 			Params: ToEthParams(x.Params),
 			State:  ToEthState(x.State),
-			Sigs:   x.Sigs,
+			Sigs:   sigsToByteArrays(x.Sigs),
 		}
 	}
 	return
 }
 
+func sigsToByteArrays(signs []wallet.Sig) [][]byte {
+	byteArrays := make([][]byte, len(signs))
+	var err error
+	for i := range signs {
+		if signs[i] != nil {
+			byteArrays[i], err = signs[i].MarshalBinary()
```

Comment:
Same comments as above (regarding concerns of `MarshalBinary` vs byte representation for a contract call) applies here.